index.html

```diff
@@ -26,13 +26,14 @@
     <section class="home" id="home">
 
       <!-- home -->
-      <div class="home">
+      <div class="grediant">
```

Comment:
We can not change the predefined classes name.

Reply:
done